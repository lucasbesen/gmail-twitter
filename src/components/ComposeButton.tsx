import React from 'react';
import styled from 'styled-components';

const Button = styled.button`
  width: 50%;
  height: 45px;
  border: 1px solid #eee;
  border-radius: 25px;
  font-size: 15px;
  box-shadow: 0px 1px 3px 0px #d3d3d3;
  margin: 10px;
  cursor: pointer;

  :hover {
    box-shadow: 0px 1px 10px 0px #d3d3d3;
  }
`;

const ComposeButton = () => <Button>Compose</Button>;

export default ComposeButton;
