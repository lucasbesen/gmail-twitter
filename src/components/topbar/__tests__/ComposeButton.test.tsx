import React from 'react';
import renderer from 'react-test-renderer';

import ComposeButton from '../ComposeButton';

it('should render ComposeButton without error', () => {
  const component = renderer.create(<ComposeButton />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
