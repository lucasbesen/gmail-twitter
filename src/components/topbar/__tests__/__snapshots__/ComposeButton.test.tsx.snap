// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ComposeButton without error 1`] = `
<button
  className="sc-bdVaJa KESzz"
>
  Compose
</button>
`;
